.github/workflows/test-spec.yml

```diff
@@ -30,6 +30,14 @@ jobs:
           - version: 2.16.0
           - version: 2.16.0
             tests: plugins/index_state_management
+          - version: 1.3.17
+            admin_password: admin
+            tests: plugins/security
+          - version: 2.0.0
+            admin_password: admin
+            tests: plugins/security
```

Comment:
Are these APIs different in 1.3.x, 2.0 and 2.16? If they aren't very different only leave the latest, 2.16.